src/hs25.jl

```diff
@@ -0,0 +1,24 @@
+# HS problem 25 in NLS format
+#
+#   Source:
+#   W. Hock and K. Schittkowski,
+#   Test examples for nonlinear programming codes,
+#   Lecture Notes in Economics and Mathematical Systems 187,
+#   Springer Verlag Berlin Heidelberg, 1981
+#   10.1007/978-3-642-48320-2
+#
+# A. S. Siqueira, Curitiba/BR, 04/2018.
+
+export hs25
+
+"Hock-Schittkowski problem 25 in NLS format"
+function hs25(;nequ = 99)
+
+  u = [25 + (-50 * log(0.01i))^(2/3) for i = 1:nequ]
+  F(x) = [-0.01i + exp(-(u[i] - x[2])^x[3] / x[1]) for i = 1:nequ]
```

Comment:
I don't have the book with me to check. Are there really complex numbers here?

Reply:
You mean `0.01i`? The `i` is an index.

Reply:
Could we write i/100 instead?

Reply:
Done.

Reply:
Thank you!